src/clj/rems/api/services/dependencies.clj

```diff
@@ -37,12 +37,16 @@
               [(:organization res)])]
      {:from {:resource/id (:id res)} :to dep})
 
-   (for [cat (catalogue/get-localized-catalogue-items {:archived true})
-         dep [{:form/id (:formid cat)}
-              {:resource/id (:resource-id cat)}
-              {:workflow/id (:wfid cat)}
-              {:organization/id (:organization cat)}]]
-     {:from {:catalogue-item/id (:id cat)} :to dep})
+   (flatten
+    (for [cat (catalogue/get-localized-catalogue-items {:archived true :expand-catalogue-data? true})
+          dep [{:form/id (:formid cat)}
+               {:resource/id (:resource-id cat)}
+               {:workflow/id (:wfid cat)}
+               {:organization/id (:organization cat)}]]
+      (into [{:from {:catalogue-item/id (:id cat)} :to dep}]
+            (mapv (fn [category]
+                    {:from {:catalogue-item/id (:id cat)}
+                     :to (select-keys category [:category/id])}) (:categories cat)))))
```

Comment:
not sure if this is the most logical place to define dependency `from catalogue to category`, but sort of convenient because catalogue items are already iterated here